### DISCUSSION TITLE
"Branches" settings in organization?

### DISCUSSION THREAD

**Discussion by Xwilarg** [community/community#23929] [New to GitHub]
<p>Hi,</p>
<p>In GitHub repositories you can add branch protection by going in the repository settings then “Branches”, is there a way to do that in an organization settings so it applies to all repositories in it?</p>
<p>Thanks in advance,</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi there!<br>
At this time, there is no feature for organizational level branch protections. There may be a way to accomplish something like this with Actions, however.</p>
<p>We’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.<br>
You can do so by <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request.</p>
<p>You can also keep an eye on our <a href="https://github.blog/changelog/">Changelog</a> as well as the <a href="https://github.com/github/roadmap">GitHub public roadmap</a> to see features we are working on.</p>
